src/main.cpp

```diff
@@ -14,11 +15,66 @@
 #include "logger/logger.hpp"
 #include "version.hpp"
 
+#if defined(DEBUG)
+extern "C" {
+
+#include <storage.h>
+#include <xen_dom_mgmt.h>
+#include <xrun.h>
+
+char json_msg[] = "{"
+	 "\"ociVersion\" : \"1.0.1\", "
+		"\"vm\" : { "
+			"\"hypervisor\": { "
+			"\"path\": \"xen\", "
+			"\"parameters\": [\"pvcalls=true\"] "
+		"}, "
+		"\"kernel\": { "
+			"\"path\" : \"unikernel.bin\", "
+			"\"parameters\" : [ \"port=8124\", \"hello world\" ]"
+		"}, "
+		"\"hwconfig\": { "
+			"\"devicetree\": \"uni.dtb\" "
+		"} "
+	"} "
+"}";
+
+extern char __img_unikraft_start[];
+extern char __img_unikraft_end[];
+extern char __dtb_unikraft_start[];
+extern char __dtb_unikraft_end[];
+
+extern void init_root();
+
+static void prepare_configs()
+{
+	int ret;
+
+	init_root();
+
+	printk("dtb = %p\n", (void *)__dtb_unikraft_start);
+	ret = write_file("/lfs", "config.json", json_msg, sizeof(json_msg));
```

Comment:
(question) do you need to continue execution in case when one of `write_file` calls fail?